gateway/server.go

```diff
@@ -912,6 +941,26 @@ func handleCORS(router *mux.Router, spec *APISpec) {
 	}
 }
 
+func getCORSHandler(spec *APISpec) func(h http.Handler) http.Handler {
+
+	if spec.CORS.Enable {
```

Comment:
reverse the condition and early exit on spec == nil || !spec.CORS.Enable, decrease indentation